views/shared/_snowplow.ejs

```diff
@@ -0,0 +1,30 @@
+<!-- Snowplow starts plowing -->
+<script type="text/javascript">
+
+    ;(function(p,l,o,w,i,n,g){if(!p[i]){p.GlobalSnowplowNamespace=p.GlobalSnowplowNamespace||[];
+    p.GlobalSnowplowNamespace.push(i);p[i]=function(){(p[i].q=p[i].q||[]).push(arguments)
+    };p[i].q=p[i].q||[];n=l.createElement(o);g=l.getElementsByTagName(o)[0];n.async=1;
+    n.src=w;g.parentNode.insertBefore(n,g)}}(window,document,"script","//d36fqcuygdrd4y.cloudfront.net/BuKMCyKUvvyXZkMi44LjI.js","snowplow"));
+
+    window.snowplow('newTracker', 'co', "<%- process.env.NODE_ENV === 'production' ? 'track.datacamp.com' : 'com-datacamp.mini.snplow.net' %>", {
+      platform: 'web',
+      post: true,
+      discoverRootDomain: true,
+      contexts: {
+        webPage: true,
+        performanceTiming: true
+      }
+    });
+    window.snowplow('enableActivityTracking', 10, 10); // Ping every 10 seconds after 10 seconds
+    window.snowplow('enableLinkClickTracking');
```

Comment:
For this feature, it works for the links in the modal. Just wanted to mention here that this method actually uses `document.links` to find all links in the DOM. Because the modal is already there from the start, this works fine on this occasion. But watch out when you dynamically add links to the DOM, they won't be tracked by this.

Reply:
This is very good to know, especially as some of these experiments will be dynamically injecting a whole bunch of HTML. Thanks!